packages/components/src/color-palette/test/index.tsx

```diff
@@ -236,24 +239,25 @@ describe( 'ColorPalette', () => {
 		);
 
 		// Confirm the correct color name, color value, and button label are used
-		expect( screen.getByText( EXAMPLE_COLORS[ 0 ].name ) ).toBeVisible();
-		expect( screen.getByText( EXAMPLE_COLORS[ 0 ].color ) ).toBeVisible();
+		expect(
+			screen.getByText( colorName, { selector: 'span' } )
+		).toBeVisible();
+		expect( screen.getByText( colorCode ) ).toBeVisible();
```

Comment:
This could be further clarified, similar to how we did for the color name above. For example, you could do:
```suggestion
		expect(
			screen.getByText( colorCode, {
				selector: '.components-color-palette__custom-color-value',
			} )
		).toBeVisible();
```